docs/articles/baseplatform/reproducible-yocto-builds.rst

```diff
@@ -0,0 +1,130 @@
+Reproducible Yocto builds
+=========================
+
+Intro
+-----
+
+This document describes how to configure a Yocto build so that it can be reproduced,
+and how to reproduce such build.
+The reproduction requires that additional artifacts are archived.
+
+A bitbake recipe specifies where the source code of a package should be downloaded from.
```

Comment:
This still doesn't fit in the Intro section. How about putting all of this AUTOREV stuff into a new header called "The problem with AUTOREV" or something like that?